server/coordinator.go

```diff
@@ -234,6 +234,7 @@ func (c *coordinator) addOperator(op Operator) bool {
 		log.Infof("coordinator: add operator %+v with higher priority, remove operator: %+v", op, old)
 	}
 
+	c.histories.add(regionID, op)
```

Comment:
I need to know whether the operator running, finish or timeout. so we need to add it in here.